Update OSDM-online-api-v1.1.0-rc.yml

eliparmi opened the pull request:
ability to propose products from different providers with combination constraints:
- in POST trip-offers-search added ability to specify a 'requestedSection', using either stations or connection points, complementing the origin/destination. Providers can then understand which sub-part of the overall trip they are requested to price
- added combination tags in the product, to let providers specify whether a product can only be sold in conjunction with a product of another provider.

nixxus1030 commented on specification/v1.1.0-rc/OSDM-online-api-v1.1.0-rc.yml:
```diff
@@ -3095,6 +3118,21 @@ components:
           type: string
         carrierConstraintText:
           type: string
+        combinationTags:
```
at offer level,  I called this tripTag, since the returnTag is a kind of combinationtag as well, IMO. But I am fine with the combinationTag naming. However I wonder: the tag on product level is IMO more as an infomation, and the one(s) on offer level those to be actually used when putting an offer together for the the upper level. What do you think ?

nixxus1030 commented on specification/v1.1.0-rc/OSDM-online-api-v1.1.0-rc.yml:
```diff
@@ -3095,6 +3118,21 @@ components:
+              combinationTag:
+                type: string
+              sellableStandalone:
+                description: >-
```
This addition does make sense: with the current specs we cannot differentiate if a product must absolutely be sold in conjunction with another or not

nixxus1030 reviewed (changes requested):
The point on requesting sub-section offers needs to be addressed. Other points are more a matter of alignement

schlpbch reviewed (changes requested):
(2): Remove offerClusters as it is redundant to coveredSegmentIndexes.
(2): A combinationTag on offer does not provide additional value --> not added. Combination of offers is controlled on product level.
(2): Rework of /trip-offer-collection
(2): Remodel tripId to tripHash (remove uuid type)
(2): Collection of segments in trip search
(2): New Boolean flag afterSaleByDistributorsOnly on BookingRequest.selectedOffers.selectedOfferIds structure
(2): Combination criteria needs to be respected in after-sale processes (basically partial-refund on one part leads to exchange of the other part).

schlpbch reviewed (approved):
Mille Gracie Elisa!

eliparmi commented:
Dear all,
I believe I've addressed all the points discussed this morning, but the tripHash. Can I leave this last item to you? I don't clearly see what has to be done.
Many thanks,
Elisa
